decrypt.go

```diff
@@ -44,76 +40,30 @@ type MessageKeyInfo struct {
 	NumAnonReceivers int
 }
 
-func (ds *decryptStream) Read(b []byte) (n int, err error) {
-	for n == 0 && err == nil {
```

Comment:
This loops seems dangerous -- if b were empty, then this seems like it would infinite-loop.